tests/qa_automation/xfstests_device.pm

```diff
@@ -24,35 +24,38 @@ sub dev_create_partition {
         $test_fs_type = "xfs";
         print "warning: TEST_FS_TYPE not configured, test defaultly set it to xfs.";
     }
-    assert_script_run("parted -l -m 2>&1");
-    my $cmd               = "parted -l -m 2>&1| awk -F \':\' \'{if(\$5 == \"xfs\") print \$1}\'";
+    type_string "df -h\n";
+    type_string "parted --script --machine -l 2>&1\n";
+    my $cmd               = "parted --script --machine -l 2>&1| awk -F \':\' \'{if(\$5 == \"xfs\") print \$1}\'";
     my $test_partition_id = script_output($cmd, 10);
     my $test_partition    = "/dev/vda" . $test_partition_id;
 
     # seperate xfs partition(/home) into two same size xfs
-    $cmd = "parted -l -m 2>&1| awk -F \':\' \'{if(\$5 == \"xfs\") print \$2}\'";
+    $cmd = "parted --script --machine -l 2>&1| awk -F \':\' \'{if(\$5 == \"xfs\") print \$2}\'";
     my $partition_begin = script_output($cmd, 10);
     my $extendpartition_begin = $partition_begin;
-    $cmd = "parted -l -m 2>&1| awk -F \':\' \'{if(\$5 == \"xfs\") print \$3}\'";
+    $cmd = "parted --script --machine -l 2>&1| awk -F \':\' \'{if(\$5 == \"xfs\") print \$3}\'";
     my $partition_end       = script_output($cmd, 10);
     my $extendpartition_end = $partition_end;
     my $partition_cut_point = $self->get_cut_point($partition_begin, $partition_end, 2);
     assert_script_run("umount " . $test_partition);
     type_string "parted /dev/vda\n";
     type_string "rm " . $test_partition_id . "\n", 5;
-    type_string "mkpart extended " . $partition_begin . " " . $partition_end . "\n", 5;
+    #type_string "mkpart extended " . $partition_begin . " " . $partition_end . "\n", 5;
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
@okurz Agree with you, will update it.